DfE.FindInformationAcademiesTrusts/Services/Export/Builders/PipelineAcademiesBuilder.cs

```diff
@@ -0,0 +1,75 @@
+﻿using DfE.FindInformationAcademiesTrusts.Data.Repositories.Trust;
+using DfE.FindInformationAcademiesTrusts.Services.Academy;
+
+namespace DfE.FindInformationAcademiesTrusts.Services.Export.Builders
+{
+    public class PipelineAcademiesBuilder(string sheetName = "Pipeline Academies") : ExportBuilder(sheetName)
+    {
+        public PipelineAcademiesBuilder WriteTrustInformation(TrustSummary? trustSummary)
+        {
+            WriteTrustInformation<PipelineAcademiesBuilder>(trustSummary);
+            return this;
+        }
+
+        public PipelineAcademiesBuilder WriteHeadersForPreAdvisory()
+        {
+            AddRow();
+            WriteHeaders(["Pre-advisory academies"]);
+            WriteHeaders([ExportHelpers.SchoolName, ExportHelpers.Urn, ExportHelpers.AgeRange, ExportHelpers.LocalAuthority, "Project type", "Proposed conversion or transfer date"]);
+
+            return this;
+        }
+
+        public PipelineAcademiesBuilder WriteHeadersForPostAdvisory()
+        {
+            AddRow();
+            WriteHeaders(["Post-advisory academies"]);
+            WriteHeaders([ExportHelpers.SchoolName, ExportHelpers.Urn, ExportHelpers.AgeRange, ExportHelpers.LocalAuthority, "Project type", "Proposed conversion or transfer date"]);
+          
+            return this;
+        }
+
+        public PipelineAcademiesBuilder WriteHeadersForFreeSchools()
+        {
+            AddRow();
+            WriteHeaders(["Free schools"]);
+            WriteHeaders([ExportHelpers.SchoolName, ExportHelpers.Urn, ExportHelpers.AgeRange, ExportHelpers.LocalAuthority, "Project type", "Provisional opening date"]);
+            
+            return this;
+        }
+
+        public PipelineAcademiesBuilder WriteRows(AcademyPipelineServiceModel[] pipelineAcademies)
+        {
+            var orderedPipelineAcademies = pipelineAcademies.OrderBy(a => a.EstablishmentName).ToArray();
+
+            foreach (var details in orderedPipelineAcademies)
+            {
+                GeneratePipelineAcademyRow(details);
+                AddRow();
```

Comment:
Calling `AddRow` after `GeneratePipelineAcademyRow` sounds like we're putting a blank row between each line. We could instead pop a `CurrentRow++` at the end of the `GeneratePipelineAcademyRow` method